c/meterpreter/source/extensions/winpmem/winpmem.h

```diff
@@ -0,0 +1,166 @@
+#include "windows.h"
+#include "stdio.h"
+#include "tchar.h"
+#include <stdint.h>
+#include "elf.h"
+#include <stdio.h>
+#include <stdarg.h>
+#include <varargs.h>
+
+// Executable version.
+#define PMEM_VERSION "1.6.2"
+#define PMEM_DEVICE_NAME "pmem"
+#define PMEM_SERVICE_NAME TEXT("pmem")
+
+static TCHAR version[] = TEXT(PMEM_VERSION) TEXT(" ") TEXT(__DATE__);
+
+// These numbers are set in the resource editor for the FILE resource.
+#define WINPMEM_64BIT_DRIVER 104
+#define WINPMEM_32BIT_DRIVER 105
+#define WINPMEM_FCAT_EXECUTABLE 106
```

Comment:
This resource does not exist. Is fcat.exe supposed to build from source, or does the blob need to be included here.